src/EnergyPlus/UnitarySystem.cc

```diff
@@ -11665,46 +11606,26 @@ namespace UnitarySystems {
             }
         } break;
         case DataHVACGlobals::Coil_CoolingWaterToAirHPSimple: {
-            // WSHP coils do not have a PLF curve and do not adjust fan performance
-            if (PartLoadRatio > 0.0 && this->m_WSHPRuntimeFrac > 0.0 && this->m_FanOpMode == DataHVACGlobals::CycFanCycCoil &&
-                this->m_sysType != SysType::PackagedWSHP) {
-                state.dataHVACGlobal->OnOffFanPartLoadFraction = PartLoadRatio / this->m_WSHPRuntimeFrac;
-            }
-
```

Comment:
Now WSHP coils do have a PLF curve! Fan performance is based on the coil runtime factor now.